codelists/forms.py

```diff
@@ -14,7 +51,8 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         self.helper = FormHelper()
-        self.helper.add_input(Submit("submit", "Submit"))
+        self.helper.form_tag = False
+        self.helper.disable_csrf = True
```

Comment:
Why do we need to do this?

Reply:
We want to keep the benefit of the Crispy `FormHelper` instance but also to end up with an HTML form comprised of 1 Form and 2 FormSet instances.

* `form_tag = False` removes the wrapping `<form>` element which is step 1.
* `disable_csrf = True` removes the `{% csrf_token %}` and I've just realised that's a mistake on my part from trying to add formsets initially!  Fixed this in e1a9fe1